src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -161,13 +168,10 @@ private function getMethodsToAutowire(\ReflectionClass $reflectionClass, array $
      *
      * @return array
      */
-    private function autowireMethodCalls(\ReflectionClass $reflectionClass, array $methodCalls, array $autowiredMethods)
+    private function autowireCalls(\ReflectionClass $reflectionClass, array $methodCalls, array $autowiredMethods)
     {
-        $parameterBag = $this->container->getParameterBag();
-
         foreach ($methodCalls as $i => $call) {
             list($method, $arguments) = $call;
-            $method = $parameterBag->resolveValue($method);
```

Comment:
this should not happen - we don't resolve method call in containerbuilder/phpdumpe